
Вкладений блок try
---------------------
В Java використання блоку try всередині іншого блоку try дозволено.
Це називається вкладеним блоком try.

Наприклад, внутрішній блок try може використовуватись для
обробки ArrayIndexOutOfBoundsException, у той час як
зовнішній блок try може обробляти ArithmeticException
(ділення на нуль).

Навіщо використовувати вкладений блок?
Іноді може виникнути ситуація, коли частина блоку може
викликати одну помилку, а весь блок може викликати іншу помилку.
У таких випадках обробники винятків мають бути вкладеними.
